requirements/requirements.txt

```diff
@@ -1,10 +1,11 @@
 # graph_altimeter dependencies.
 gremlinpython==3.4.12
+policyuniverse==1.4.0.20220110
 # Altimeter fork.
 # TODO: revert to upstream once fixes are applied.
 git+https://github.com/manelmontilla/altimeter.git@dev
 
 # neptune_python_utils dependencies.
 boto3==1.20.44
 backoff==1.11.1
-aiohttp==3.8.1
```

Comment:
Can we pin a specific version?

Reply:
Yep, knowing the precedents, it makes a lot of sense...